corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -128,26 +131,28 @@
     {% endblocktrans %}
 </div>
 {% endif %}
-<div class="pull-right">
-    {% if auth_context.authenticated %}
-        <i class="icon-lock"></i>
-        {% if user_info.id != auth_user_info.id %}
-        {% blocktrans with auth_user=auth_user_info|pretty_doc_info user=user_info|pretty_doc_info %}
-            Submitted by {{ auth_user }} on behalf of {{ user }}
-        {% endblocktrans %}
+<div id="submit-info">
+    <div class="pull-right">
```

Comment:
hmm, i'd rather not move it since it's been like this for a while and then it goes inside the tabs. can try removing the other one too though.